contracts/test/auth.ts

```diff
@@ -123,7 +128,7 @@ describe('Auth', function () {
       siweStr,
       await erc191sign(siweStr, account),
     );
-    await expect(await siweAuthTests.testVerySecretMessage(bearer)).to.be.equal(
+    expect(await siweAuthTests.testVerySecretMessage(bearer)).to.be.equal(
```

Comment:
Is there a difference between to.be.equal and to.be.reverted for sync/async behavior?

Reply:
Yes, reverted takes a promise, you await the `expect`.
Otherwise, you await the result, then `expect` it.